godel_plugins/include/godel_plugins/widgets/robot_blending_widget.h

```diff
@@ -185,6 +106,10 @@ Q_OBJECT
 	bool call_select_surface_service(godel_msgs::SelectSurface::Request &req);
 	bool call_surface_detection_service(godel_msgs::SurfaceDetection& s);
 	void selected_surface_changed_callback(godel_msgs::SelectedSurfacesChangedConstPtr msg);
+	void select_motion_plan(const std::string& name, bool simulate);
+	void request_available_motions(std::vector<std::string>& plans);
+	void request_load_save_motions(const std::string& path, bool isLoad);
+	void update_motion_plan_list(const std::vector<std::string>& names);
```

Comment:
method name formatting has been brought up before in previous PRs, it'll be dealt with in another PR specific to this issue.